server/game/cards/events/01/risenfromthesea.js

```diff
@@ -14,6 +14,7 @@ class RisenFromTheSea extends DrawCard {
             handler: context => {
                 context.event.saveCard(context.target);
                 this.controller.attach(this.controller, this, context.target, 'play');
+                this.setCardType('attachment');
```

Comment:
Blanking an attached Risen would not remove the attachment cardtype, as that's not part of the textbox at that point.